hieunc-guestbook-dojo/static/js/guestbook/models/Greeting.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Created by hieunc on 25/11/2015.
+ */
+
+define([
+	"dojo/_base/declare"
+], function (declare) {
+	return declare("Greeting", null, {
```

Comment:
- hãy declare class name theo cấu trúc sau:
  - "path.to.file.ClassName"
  - ex: "guestbook.models.Greeting"
- kiểm tra lại các file khác